proxy/Main.cc

```diff
@@ -908,7 +908,7 @@ check_fd_limit()
   if (fds_throttle > fds_limit + THROTTLE_FD_HEADROOM) {
     int new_fds_throttle = fds_limit - THROTTLE_FD_HEADROOM;
     if (new_fds_throttle < 1) {
-      MachineFatal("too few file descritors (%d) available", fds_limit);
+      ink_abort("too few file descritors (%d) available", fds_limit);
```

Comment:
``Fatal`` would be better than ``ink_abort`` since it formats its message to the right log sinks before actually aborting.